doc/source/examples/sharp-immersed-boundary/inclined-3d-composite-mixer-with-pbt-impeller/inclined-3d-composite-mixer-with-pbt-impeller.rst

```diff
@@ -0,0 +1,284 @@
+
+.. code-block:: text
+
+    subsection orientation
+        set Function expression =atan2(0.707106781186548*sin(pi/4)*sin(6.28318530717959*t) - 0.707106781186548*sin(6.28318530717959*t)*cos(pi/4), (0.5 - 0.5*cos(6.28318530717959*t))*sin(pi/4) + (0.5*cos(6.28318530717959*t) + 0.5)*cos(pi/4));asin((0.5 - 0.5*cos(6.28318530717959*t))*cos(pi/4) + (0.5*cos(6.28318530717959*t) + 0.5)*sin(pi/4));atan2(-0.707106781186548*sin(6.28318530717959*t), -(0.5 - 0.5*cos(6.28318530717959*t))*sin(pi/4) + (0.5*cos(6.28318530717959*t) + 0.5)*cos(pi/4))
+    end
+
+The parameters used to define the impeller are based on the :doc:`../3d-composite-mixer-with-pbt-impeller/3d-composite-mixer-with-pbt-impeller` example and are given as follows:
+
+
+.. code-block:: text
+
+    subsection particles
+      set number of particles                     = 1
+      set assemble Navier-Stokes inside particles = false
+      subsection extrapolation function
+        set stencil order = 2
+        set length ratio  = 3
+      end
+      subsection local mesh refinement
+        set initial refinement                = 6
+        set refine mesh inside radius factor  = 0
+        set refine mesh outside radius factor = 1.25
+      end
+      subsection output
+        set enable extra sharp interface vtu output field = true
+      end
+      subsection particle info 0
+        subsection position
+          set Function expression = 0;0;0
+        end
+        subsection velocity
+          set Function expression = 0;0;0
+        end
+        subsection orientation
+          set Function expression =atan2(0.707106781186548*sin(pi/4)*sin(6.28318530717959*t) - 0.707106781186548*sin(6.28318530717959*t)*cos(pi/4), (0.5 - 0.5*cos(6.28318530717959*t))*sin(pi/4) + (0.5*cos(6.28318530717959*t) + 0.5)*cos(pi/4));asin((0.5 - 0.5*cos(6.28318530717959*t))*cos(pi/4) + (0.5*cos(6.28318530717959*t) + 0.5)*sin(pi/4));atan2(-0.707106781186548*sin(6.28318530717959*t), -(0.5 - 0.5*cos(6.28318530717959*t))*sin(pi/4) + (0.5*cos(6.28318530717959*t) + 0.5)*cos(pi/4))
+        end
+        subsection omega
+          set Function expression = -1*pi*2*sqrt(2)/2;0;-1*pi*2*sqrt(2)/2
+        end
+        set type            = composite
+        set shape arguments = impeller.composite
+      end
+    end
+
+The only noticeable differences are that the initial refinement and the refinement zone are adjusted respectively to 6 and 0 to 1.25 reference length. These values are chosen to guarantee that the refinement zone is big enough to cover the motion of the impeller and avoid interaction of the hanging nodes with the sharp immersed boundary constraints.
+
+--------
+Results
+--------
+
+In the following image the velocity field obtained with this example after 1 second can be observed:
+
+.. image:: images/inclined-impeller_flow_1_sec.png
+   :alt: flow_1sec
+   :align: center
+   :name: velocity_field_norm
+   
```

Comment:
Would be nicer to also make an animation we can put online. Why not make one?
It won't take much time and will make the example cooler.

Reply:
I will create it soon during the hackaton and add it after.